libqfieldsync/project_checker.py

```diff
@@ -289,6 +294,36 @@ def check_project_is_dirty(self) -> Optional[FeedbackResult]:
         else:
             return None
 
+    def check_for_conflicting_base_filenames(self) -> Optional[FeedbackResult]:
+        conflicting_files: List[Path] = []
```

Comment:
```suggestion
        conflicting_files: list[Path] = []
```

Shouldn't we prefer the lower case typing ?

Reply:
Thanks for the reminder, yes, indeed should be on lowercase!

Reply:
Not in (lib)qfieldsync, because here we might run on ancient python version, e.g. on Debian or windows builds.